docs/source/contributing/documentation.rst

```diff
@@ -3,8 +3,80 @@ Adding Documentation
+* Examples don't require the ``from manim import *`` statement, this will be added automatically when the docs are build```
+
+* There should be a balance of animated and non-animated examples.
+
+- As manim makes animations, we can include lots of animated examples; however, our RTD has a maximum 15 minutes to build. Animated examples should only be used when necessary, as last frame examples render faster.
+
+- Lots of examples (e.g. size of a plot-axis, setting opacities, making texts, etc.) will also work as images. It is a lot more convenient to see the end product immediately instead of waiting for an animation to reveal it.
+
+* Examples should not have comments in the code. Explanations should be added outside of the example code.
+
+* Please ensure the examples run on the current master when you contribute an example.
+
+How examples are structured
+---------------------------
+
+* Examples can be organized into chapters and subchapters.
+
+- When you create examples, the beginning example chapter should focus on only one functionality. When the functionality is simple, multiple ideas can be illustrated under a single example.
+
+- As soon as simple functionalities are explained, the chapter may include more complex examples which build on the simpler ideas.
+
+Writing examples
+~~~~~~~~~~~~~~~~
+
+When you want to add/edit examples, they can be found in the ``docs/source/`` directory, or directly in the manim source code (e.g. ``manim/mobject/mobject.py``). The examples are written in ``rst`` format and use the manim directive (see :mod:`~.manim_directive` ), ``.. manim::``. Every example is in its own block, and looks like this:
+
+ .. code:: rst
+
+    Formulas
+
+    ========
+
+    .. manim:: Formula1
+        :save_last_frame:
+
+        class Formula1(Scene):
+            def construct(self):
+                t = MathTex(r"\int_a^b f'(x) dx = f(b)- f(a)")
+                self.add(t)
+                self.wait(1)
+
+In the building process of the docs, all ``rst`` files are scanned, and the manim directive (``.. manim::``) blocks are identified as scenes, that will be run by the current version of manim.
+Here is the syntax:
+
+* ``.. manim:: [SCENE_NAME]`` has no indentation and ``SCENE_NAME`` refers to the name of the class below.
+
+* The flags are followed in the next line (no blank line here!), with the indention level of one tab.
+
+All possible flags can be found at :mod:`~.manim_directive`
+
+In the example above, ``Formula1`` followed by ``.. manim::`` and is the scene that will be rendered, so in the python code the class must have the same name: ``class Formula1(Scene)``
+
+.. note::
+
+   Sometimes, when you reload an example in your browser, it has still the old website somewhere in its cache. If this is the case, delete the website cache, or open a new `incognito tab`` in your browser, then the latest docs should be shown. If this still doesn't work, you may need to delete the contents of ``docs/source/references``.
```

Comment:
The last part specifically is only relevant for locally built documentation, it should say so.

Reply:
I also feel like this is a bit out of place... Up until this point, the reader might not have read anything about building the documentation locally and would likely not remember this note when it is mentioned. The only mention of `make html` is in the `Manim Development Process` section way at the bottom:
> * Documentation: We also build a version of the documentation corresponding to your pull request. Make sure not to introduce any Sphinx errors, and have a look at the built HTML files to see whether the formatting of the documentation you added looks as you intended. You can build the documentation locally by running make html from the docs directory.

Perhaps this bullet point should be moved to the documentation section? Or a more in-depth guide on building the documentation locally is prerequisite information?

Reply:
I am thinking the we should have a section explaining it in depth

Reply:
Same; however, this should be done in a separate PR. For now, I still think the mention of it being locally built is necessary.

Reply:
It is mentioned I think.